package.json

```
@@ -86,15 +86,15 @@
   },
   "types": "types/runtime/index.d.ts",
   "scripts": {
-    "test": "mocha --exit",
-    "test:unit": "mocha --require sucrase/register --recursive src/**/__test__.ts --exit",
-    "quicktest": "mocha",
+    "test": "npm run test:integration && npm run test:unit",
```

should we put the unit tests first since they'll presumably run faster?
```suggestion
    "test": "npm run test:unit && npm run test:integration",
```